python/cudf/cudf/core/column_accessor.py

```diff
@@ -341,6 +341,58 @@ def _pad_key(self, key: Any, pad_value="") -> Any:
             key = (key,)
         return key + (pad_value,) * (self.nlevels - len(key))
 
+    def replace_level_values(
+        self, mapping: dict or callable, level: int or None
+    ):
```

Comment:
```suggestion
    ) -> ColumnAccessor:
```

Reply:
When adding type annotations, also add the return type.

Reply:
not sure what you mean by this, can you please clarify?